### PR TITLE
Enchance/meshery integrations honeycomb

### LINKED ISSUE #4349: Enhance the Integrations honeycomb cells with a radial gradient
### Current Behavior
These honeycomb cells use a linear gradient:

Screenshot 2023-06-06 at 6 48 23 PM

### Desired Behavior
Change from linear gradient by enhance the Integrations honeycomb cells with a radial gradient.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Description**
Meshery/Meshery-integrations  : Enchance the Meshery Integrations Honeycomb by using radial gradient as background .

This PR fixes #4349

**Notes for Reviewers**
- Added A new variable to app themeStyle  "radialToWhite" 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 02c15947463a036c01dbfedda63c8c74bf2ac81e at: https://6481dad12712ea00d1cf87ea--layer5.netlify.app

File: src/theme/app/themeStyles.js [hunk 1]
```diff
@@ -395,6 +396,7 @@ export const darktheme = {
   linearToLinear2: "linear-gradient(61.3deg, rgba(0, 179, 159, 0.2) -26.19%, rgba(18, 18, 18, 0.2) 40.19%), linear-gradient(303.45deg, rgba(0, 179, 159, 0.2) 0%, rgba(18, 18, 18, 0.2) 37.23%), linear-gradient(360deg, #00B39F -11.84%, #121212 63.36%)",
   linearToGreen: "linear-gradient(90.2deg, rgba(0, 0, 0, 0.2) 0.16%, rgba(0, 179, 159, 0.74) 99.81%)",
   linearToWhite: "linear-gradient(to right, #666666, #444444)",
+  radialToWhite: "radial-gradient(circle, rgba(102,102,102,1) 0%, rgba(68,68,68,1) 100%)",
```
Comment (leecalcote) [inline src/theme/app/themeStyles.js:399]:
```suggestion
  radialToWhite: "radial-gradient(circle, rgba(102,102,102,1) 0%, rgba(28,28,28,1) 100%)",
```

**Review by leecalcote — APPROVED** [review]
Looking good, @aabidsofi19 👍

**Comment by leecalcote** [comment]
Thanks, @aabidsofi19 🙂

**Pull request merged**